src/app/forms-builder/components/dynamic-fields/dynamic-form.component.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  AfterViewChecked,
+  AfterViewInit,
+  Component,
+  Input,
+  OnInit,
+  ViewChild,
+  ViewChildren
+} from '@angular/core';
+import { FormGroup, FormBuilder } from '@angular/forms';
+import { fromEvent } from 'rxjs';
+import { Store } from '@ngrx/store';
+
+import { FieldConfig } from '../../../core/interfaces/field.interface';
+import { FormsBuilderContentState } from '../../../core/state/reducers';
+import { FormsBuilderService } from '../../services/forms-builder.service';
+import { DropAreaChangeIndexAction } from '../../../core/state/actions/dropAreaActions';
+import {takeUntil} from "rxjs/operators";
+
+
+@Component({
+  selector: 'app-dynamic-form',
+  templateUrl:'./dynamic-form.component.html',
+  styleUrls:[`./dynamic-form.component.scss`]
+})
+
+export class DynamicFormComponent implements OnInit,AfterViewInit,AfterViewChecked {
+
+  public form: FormGroup;
+  public changeContent$:any
+
+  @Input() public fields: FieldConfig[] = [];
+  @Input() public isStyleInput:boolean
+  @Input() public isFormActive:boolean
+  @Input() public items:any
+  @Input() public selectedIndex:any
```

Comment:
add types, spaces, semi-columns 
```suggestion
  @Input() public isStyleInput: boolean;
  @Input() public isFormActive: boolean;
  @Input() public items: add type;
  @Input() public selectedIndex: add type;
```